main_test.go

```diff
@@ -90,6 +90,10 @@ func TestEndpoints(t *testing.T) {
 		{"/search?type=input&prerelease=true", "/search", "search-input-packages.json", searchHandler(testLogger, indexer, testCacheTime)},
 		{"/search?type=input&package=integration_input&prerelease=true", "/search", "search-input-integration-package.json", searchHandler(testLogger, indexer, testCacheTime)},
 		{"/search?type=integration&package=integration_input&prerelease=true", "/search", "search-integration-integration-package.json", searchHandler(testLogger, indexer, testCacheTime)},
+		{"/search?capabilities=observability,security&prerelease=true", "/search", "search-prerelease-capabilities-observability-security.json", searchHandler(testLogger, indexer, testCacheTime)},
+		{"/search?capabilities=none&prerelease=true", "/search", "search-prerelease-capabilities-none.json", searchHandler(testLogger, indexer, testCacheTime)},
+		{"/search?spec.min=1.1.0&spec.max=2.10.0&prerelease=true", "/search", "search-spec-min-1.1.0-max-2.10.0.json", searchHandler(testLogger, indexer, testCacheTime)},
```

Comment:
I see I didn't copy this to the description of the issue, sorry. In the original proposal we thought about using only <major>.<minor> (as equivalent to <major>.<minor>.0), so if a version of Kibana supports 2.0.0, it also supports 2.0.1 or any other 2.0.X version. Patch versions shouldn't include differences that break compatibility, or new features.

So the API would work like this:
```suggestion
		{"/search?spec.min=1.1&spec.max=2.10&prerelease=true", "/search", "search-spec-min-1.1.0-max-2.10.0.json", searchHandler(testLogger, indexer, testCacheTime)},
```

Reply:
Ok, it makes sense.

Updated the tests accordingly with that.

I also added a validation to check that versions in the query parameter are just  `major.minor` (added also a test for that).